libkmod/libkmod-file.c

```diff
@@ -75,15 +75,41 @@ struct kmod_elf *kmod_file_get_elf(struct kmod_file *file)
 	return file->elf;
 }
 
+static inline ssize_t pread_safe(int fd, char *buf, size_t buflen, off_t offset)
```

Comment:
it's almost a 1:1 copy of read_str_safe(). Better to add this as pread_str_safe() in util.c

Reply:
It's copied from there. ;) Adjusted